source/Projectile.js

```diff
@@ -68,7 +68,7 @@ export default class Projectile extends Pixi.Sprite {
           }
       }
 
-      if(this.despawn == true) {
+      if(this.mineDespawn == true) {
```

Comment:
This strikes me as a bad way to do this, it means you're checking an attribute that is only defined for mines and superMines. Trying to compare an undeclared variable raised my hackles, and is usually undefined behavior. If this code only executes if you're dealing with a mine or superMine, why don't you just check projectileType instead? Shouldn't this work just fine with if(this.projectileType === "mine" || this.projectileType === "superMine")